src/hooks/useFormField.tsx

```
@@ -0,0 +1,70 @@
+  const { ref: inputRef, ...inputProps } = register(name, validationRules);
+
+  const handleChange = (event: any) => {
+    if (event?.nativeEvent) {
+      inputProps.onChange(event);
+    } else {
+      setValue(name, event.target.value);
+    }
+    trigger(name);
+  };
+
+  const createCustomEvent = (value: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)